nchelpers/__init__.py

```diff
@@ -611,6 +611,10 @@ def sampling_geometry(self):
             # TODO: Check more carefully for gridded datasets?
             return 'gridded'
 
+    @property
+    def is_multi_year(self):
+        return is_multi_year_mean(self)
```

Comment:
`is_multi_year_mean` is a property too, so I think you need to omit `(self)`. Also please add test(s) for this, just dup the tests for `is_multi_year_mean`.

Reply:
Ah right, good point!